httpx/_client.py

```diff
@@ -1154,6 +1209,11 @@ async def send(
         allow_redirects: bool = True,
         timeout: typing.Union[TimeoutTypes, UnsetType] = UNSET,
     ) -> Response:
+        """
+        Send a pre-built request.
+
+        Does not merge parameters with the client.
+        """
```

Comment:
```suggestion
        """
        Send a request.

        The request is sent as-is, unmodified.

        Typically you'll want to get it from `Client.build_request()` (so that any
        client-level configuration is merged into the request), but passing
        an explicit `httpx.Request()` is supported as well.

        See also: [Request instances][0]

        [0]: /advanced/#request-instances
        """
```

Reply:
Added this with a minor rewording to the new version!